test/test-cases/functional/saic/sai-api/test_sai_api_vnet_acl_out.py

```diff
@@ -0,0 +1,96 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+
+
+class TestSaiVnetAclOut:
+
+    def test_vnet_acl_out_create(self, dpu):
+
+        commands = [
+            {
+                "name": "acl_out",
+                "op": "create",
+                "type": "SAI_OBJECT_TYPE_DASH_ACL_GROUP",
+                "attributes": [
+                    "SAI_DASH_ACL_GROUP_ATTR_IP_ADDR_FAMILY",
+                    "SAI_IP_ADDR_FAMILY_IPV4"
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values create =======")
+        pprint(results)
+
+        assert all(results), "SAI_IP_ADDR_FAMILY_IPV4 Create error"
```

Comment:
@murthyvijay Could you please check all assert messages? I see that the next 3 messages do not correspond to the SAI calls.